packages/lit-html/src/lit-html.ts

```diff
@@ -814,35 +814,19 @@ class TemplateInstance {
 /*
  * Parts
  */
-type AttributePartConstructor = {
-  new (
-    element: HTMLElement,
-    name: string,
-    strings: ReadonlyArray<string>,
-    parent: Disconnectable | undefined,
-    options: RenderOptions | undefined
-  ): AttributePart;
-};
 type AttributeTemplatePart = {
   readonly type: typeof ATTRIBUTE_PART;
   readonly index: number;
   readonly name: string;
   /** @internal */
-  readonly ctor: AttributePartConstructor;
+  readonly ctor: typeof AttributePart;
   /** @internal */
   readonly strings: ReadonlyArray<string>;
 };
 type NodeTemplatePart = {
   readonly type: typeof CHILD_PART;
   readonly index: number;
 };
-type ElementPartConstructor = {
-  new (
-    element: HTMLElement,
-    parent: Disconnectable | undefined,
-    options: RenderOptions | undefined
-  ): ElementPart;
-};
```

Comment:
After removing the casts, I had to remove `ElementPartConstructor` because that was its only use. I removed `AttributePartConstructor` above also, because its one use seems like it can be replaced with `typeof AttributePart`. WDYT?

Reply:
cc @justinfagnani

Reply:
Seems fine. As long as everything checks, we're good.